rust/src/block/mod.rs

```diff
@@ -45,15 +45,28 @@ pub struct BlockWithoutHeight {
 pub struct BlockHash(pub String);
 
 impl BlockHash {
-    pub fn from_bytes(bytes: [u8; 32]) -> Self {
-        let block_hash = unsafe { String::from_utf8_unchecked(Vec::from(bytes)) };
-        Self(block_hash)
+    pub const LEN: usize = 52;
+    pub const PREFIX: &'static str = "3N";
```

Comment:
You don't need to `&'static` prefix if it's const

Reply:
![image](https://github.com/Granola-Team/mina-indexer/assets/42081488/2815d720-d03c-4961-a4c1-8a9dbb0510b1)